contracts/p1/StRSR.sol

```diff
@@ -30,18 +29,17 @@ import "./mixins/Component.sol";
  *   across non-withdrawing stakes, while when RSR is seized it is seized uniformly from both
  *   stakes that are in the process of being withdrawn and those that are not.
  */
-// solhint-disable max-states-count
 abstract contract StRSRP1 is Initializable, ComponentP1, IStRSR, EIP712Upgradeable {
     using CountersUpgradeable for CountersUpgradeable.Counter;
     using SafeERC20Upgradeable for IERC20Upgradeable;
 
     /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
     // solhint-disable-next-line var-name-mixedcase
-    uint48 public immutable PERIOD; // {s} 1 block based on network
+    uint48 public constant PERIOD = 1; // {s} 1 second
```

Comment:
I noticed we are using this PERIOD var to divide and do calculations, but because its 1, cant we remove it all together? from all the contracts that have this PERIOD concept? (as long as it is 1 second doesnt provide much value)